modules/topology-manager-policies.adoc

```diff
@@ -8,7 +8,7 @@
 
 Topology Manager aligns `Pod` resources of all Quality of Service (QoS) classes by collecting topology hints from Hint Providers, such as CPU Manager and Device Manager, and using the collected hints to align the `Pod` resources.
 
-Topology Manager supports four allocation policies, which you assign in the `cpumanager-enabled` custom resource (CR):
+Topology Manager supports four allocation policies, which you assign in the KubeletConfig customer resource (CR) named cpumanager-enabled:
```

Comment:
Same comments as the other file about the typo and marking up the items properly